app/lib/logic/cms.rb

```diff
@@ -60,7 +60,7 @@ def cms_toolbar_intro_visible?
       end
 
       def create_cms_assets
-        self.builtin_sections.create!(title: 'Root', system_name: 'root', parent_id: nil)
+        self.builtin_sections.create!({ title: 'Root', system_name: 'root', parent_id: nil }, without_protection: true)
```

Comment:
Since all values are hardcoded here, using `without_protection` is safe. Whether `parent_id` is needed or not, will require investigation that we don't presently have time for. So for me it is safe to keep original implementation for this build.